From 06be5c4182f3caf7e962870d6326d536d0c4e7cf Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Tue, 30 Jul 2024 21:46:29 +0100
Subject: [PATCH 293/303] testsuite: fix 'dg-do-preprocess' typo

We want 'dg-do preprocess', not 'dg-do-preprocess'. Fix that.

	PR target/106828
	* g++.target/loongarch/pr106828.C: Fix 'dg-do compile' typo.
---
 gcc/testsuite/g++.target/loongarch/pr106828.C | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gcc/testsuite/g++.target/loongarch/pr106828.C b/src/gcc/testsuite/g++.target/loongarch/pr106828.C
index 190c1db71..0d13cbbd5 100644
--- a/src/gcc/testsuite/g++.target/loongarch/pr106828.C
+++ b/src/gcc/testsuite/g++.target/loongarch/pr106828.C
@@ -1,4 +1,4 @@
-/* { dg-do-preprocess } */
+/* { dg-do preprocess } */
 /* { dg-options "-mabi=lp64d -fsanitize=address" } */
 
 /* Tests whether the compiler supports compile option '-fsanitize=address'.  */
-- 
2.33.0

